test/parallel/test-fs-stat-abort-test.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+require('../common');
+const test = require('node:test');
+const assert = require('node:assert');
+const fs = require('node:fs');
+
+const filePath = './temp.txt';
+fs.writeFileSync(filePath, 'Test');
```

Comment:
This is not ok.

Reply:
many thanks for your suggestion I used tmpdir

Reply:
oh, ha, good catch. completely overlooked that!